src/packages/client/package.json

```
@@ -97,11 +98,13 @@
     "make-dir": "3.1.0",
     "mariadb": "2.5.3",
     "mssql": "6.3.1",
+    "patch-package": "^6.4.6",
     "pg": "8.5.1",
```

```suggestion
    "mssql": "6.3.1",
    "pg": "8.5.1",
```
We could remove since it's not needed?